src/modules/Elsa.Workflows.Sink/Extensions/FeatureExtensions.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using Elsa.Workflows.Sink.Features;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+public static class FeatureExtensions
+{
+    public static WorkflowSinkFeature UseInMemoryDatabase(this WorkflowSinkFeature feature, Action<MemoryWorkflowSinkPersistenceFeature>? configure = default)
+    {
+        feature.Module.Configure(configure);
+        return feature;
+    }
+    
+    public static WorkflowSinkFeature UseMassTransitServiceBus(this WorkflowSinkFeature feature, Action<MassTransitSinkTransportFeature>? configure = default)
+    {
+        feature.Module.Configure(configure);
+        return feature;
+    }
```

Comment:
I think the dependency between Workflow Sink and MassTransit should be the inverse: the Workflow Sink module provides a transport contract, but not the actual implementation. That is the role of e.g. Elsa.MassTransit, which then provides a concrete implementation of ISinkTransport. Same goes for the EF Core one. 

This way, whenever we add another sink implementation, we just add a new project without having to update the Elsa.Workflows.Sinks project itself (i.e. open/closed).